doc/man/scc-hypervisor-collector.1.md

```diff
@@ -0,0 +1,216 @@
+---
+title: SCC-HYPERVISOR-COLLECTOR
+section: 1
+header: SCC Hypervisor Collector
+date: June 2022
+---
+# NAME
+**scc-hypervisor-collector** - Collect & Upload Hypervisor details
+to SUSE Customer Center
+
+# SYNOPSIS
+
+**scc-hypervisor-collector [<optional>...]**
+
+# DESCRIPTION
+
+The **scc-hypervisor-collector** collects details that are relevant
+to SUSE Customer Center (SCC) subscription compliance tracking from
+the specified hypervisors.
+
+The collected details include the total system RAM, CPU architecture
```

Comment:
The upload action of this comment is not really covered. When does it occur, can it be disabled?

Reply:
Explicitly mentioned in updated description.

Not (yet) possible to disable - future enhancements may allow for "collect only" and "upload only" modes.